src/options/opt_accounts.cpp

```diff
@@ -0,0 +1,32 @@
+#include "opt_accounts.h"
+
+#include "psioptions.h"
+
+#include "accountmanagedlg.h"
+
+OptionsTabAccounts::OptionsTabAccounts(QObject *parent)
+: MetaOptionsTab(parent, "accounts", "", tr("Accounts"), tr("Manage accounts"), "psi/account")
+, w_(nullptr)
+, psi_(nullptr)
+{
+}
+
+OptionsTabAccounts::~OptionsTabAccounts()
+{
+}
+
+QWidget *OptionsTabAccounts::widget()
+{
+    if (w_) {
```

Comment:
This doesn't look right

Reply:
What do you mean? In other option tabs the same functions with the same if statements.

Reply:
oh. I see. I misintepreted the code. I thought it has to be `if(!w_)`